packages/typespec-ts/test/modularUnit/apiOperations.spec.ts

```diff
@@ -0,0 +1,224 @@
+import { assert } from "chai";
+import { emitModularOperationsFromTypeSpec } from "../util/emitUtil.js";
+import { assertEqualContent } from "../util/testUtil.js";
+
+describe("api operations in Modular", () => {
+  it("should handle contentTypes has binary data", async () => {
+    const tspContent = `
+    @route("/uploadFileViaBody")
+    @post op uploadFileViaBody(
+      @header contentType: "application/octet-stream",
```

Comment:
From Swagger, the payload of octet-stream would look like the below, so could you help to ensure the typespec is correct for this case?

requestBody:
  content:
    application/octet-stream:
      schema:
        type: string
        format: binary # binary data
    image/png:
      schema:
        type: string
        format: base64 # base64-encoded data

Reply:
Issue created https://github.com/microsoft/typespec/issues/2615